raiden/network/transport/matrix/transport.py

```diff
@@ -498,33 +498,32 @@ def start_health_check(self, node_address):
             # representing the target node
             self._address_mgr.refresh_address_presence(node_address)
 
-    def send_async(self, queue_identifier: QueueIdentifier, message: Message):
+    def send_message(self, message: TransportMessage, recipient: Address):
         """Queue the message for sending to recipient in the queue_identifier
 
         It may be called before transport is started, to initialize message queues
         The actual sending is started only when the transport is started
         """
+        raiden_message, queue_identifier = TransportMessage.unwrap(message)
 
         # even if transport is not started, can run to enqueue messages to send when it starts
-        receiver_address = queue_identifier.recipient
-
-        if not is_binary_address(receiver_address):
-            raise ValueError("Invalid address {}".format(pex(receiver_address)))
+        if not is_binary_address(recipient):
+            raise ValueError("Invalid address {}".format(pex(recipient)))
 
         # These are not protocol messages, but transport specific messages
-        if isinstance(message, (Ping, Pong)):
+        if isinstance(raiden_message, (Ping, Pong)):
             raise ValueError(
-                "Do not use send_async for {} messages".format(message.__class__.__name__)
+                "Do not use send_message for {} messages".format(raiden_message.__class__.__name__)
             )
 
         self.log.info(
-            "Send async",
-            receiver_address=pex(receiver_address),
-            message=message,
+            "Send message",
+            receiver_address=pex(recipient),
```

Comment:
i will probably change the logger parameter too to `recipient`

Reply:
done